src/views/v2/month/top-bar/datepicker.php

```diff
@@ -11,13 +11,15 @@
  *
  * @version 4.9.13
  *
- * @var string    $now                    The current date and time in the `Y-m-d H:i:s` format.
- * @var string    $grid_date              The current calendar grid date in the `Y-m-d` format.
- * @var string    $formatted_grid_date    The current calendar grid date in the format specified by the "Month and year
- *                                        format" option.
- * @var object    $date_formats           Object containing the date formats.
- * @var \DateTime $the_date               The Month current date object.
- * @var bool      $show_datepicker_submit Boolean on whether to show the datepicker submit button.
+ * @var string    $now                        The current date and time in the `Y-m-d H:i:s` format.
+ * @var string    $grid_date                  The current calendar grid date in the `Y-m-d` format.
+ * @var string    $formatted_grid_date        The current calendar grid date in the format specified by the "Month and
+ *                                            year format" option.
+ * @var string    $formatted_grid_date_mobile The current calendar grid date in the format specified by the "Compact
+ *                                            Date Format" option.
+ * @var object    $date_formats               Object containing the date formats.
+ * @var \DateTime $the_date                   The Month current date object.
```

Comment:
Feel free to ignore and move on (I realize it isn't new as of your change) but I'm surprised we need to `\` qualify this type. Would be just a tad cleaner / would remove a bit of line noise if we dropped the backslash.